code_folder/C0_data_processing.py

```diff
@@ -227,12 +279,34 @@ def process_all_assets(dict_values):
 class helpers:
     def define_missing_cost_data(dict_values, dict_asset):
 
-        # read timeseries with filename provided for variable costs
+        # read timeseries with filename provided for variable costs.
+        # if multiple opex_var or capex_var are given for multiple busses, it checks if any value is a timeseries
```

Comment:
```suggestion
        # if multiple opex_var or capex_var are given for multiple busses, it checks if any value is a timeseries
        # this enables using electricity price timeseries
```
Can you add an example of capex_var? Is it the example of peak demand pricing? Because then I would specify that capex_var is not a "timeseries" but a list for peak demand pricing.

Reply:
I was not thinking in any particular example. It might be not useful to make multiple busses available for capex_var.